BCnEncTests/ImageLoader.cs

```diff
@@ -25,8 +29,40 @@ public static class ImageLoader {
 			LoadTestImage("../../../testImages/cubemap/forward.png")
 		};
 
-		private static Image<Rgba32> LoadTestImage(string filename) {
+		private static Image<Rgba32> LoadTestImage(string filename)
+		{
 			return Image.Load<Rgba32>(filename);
 		}
 	}
+
+	public static class DdsLoader
+	{
+		public static DdsFile TestDecompressBc1 { get; } = LoadDdsFile("../../../testImages/test_decompress_bc1.dds");
+		public static DdsFile TestDecompressBc1A { get; } = LoadDdsFile("../../../testImages/test_decompress_bc1a.dds");
+		public static DdsFile TestDecompressBc7 { get; } = LoadDdsFile("../../../testImages/test_decompress_bc7.dds");
+		public static DdsFile TestDecompressRgba { get; } = LoadDdsFile("../../../testImages/test_decompress_rgba.dds");
+
+		private static DdsFile LoadDdsFile(string filename)
+		{
+			return DdsFile.Load(File.OpenRead(filename));
```

Comment:
FileStream is not disposed here. Same with ktx.

Reply:
Do we even need to dispose those resources? It's not like the GC won't clean everything up after the tests are done.

Reply:
I think it's just good to do. And it only adds 1 extra line to each method.